packages/shared/ReactSymbols.js

```diff
@@ -48,6 +49,7 @@ if (typeof Symbol === 'function' && Symbol.for) {
   REACT_RESPONDER_TYPE = symbolFor('react.responder');
   REACT_SCOPE_TYPE = symbolFor('react.scope');
   REACT_OPAQUE_ID_TYPE = symbolFor('react.opaque.id');
+  REACT_DEBUG_TRACE_MODE_TYPE = symbolFor('react.log_mode');
```

Comment:
A bit inconsistent

Reply:
Whoops. I forgot to rename from its initial :)